.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1 +1 @@
-distributionUrl=https://storage.googleapis.com/cloud-tools-for-java-team-kokoro-build-cache/apache-maven-3.5.0-bin.zip
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.2/apache-maven-3.5.2-bin.zip
```

Comment:
we should probabkly put this in the google cache if we need it

Reply:
Was this the cause of significant slowdowns previously? The history of this file starts off using the google cache version of this file, but we use the maven repo version in some other projects.